### ISSUE TITLE
Brain_2.5DUNet

### ISSUE THREAD

**Issue by yingmuzhi** [mehta-lab/microDL#240]
Hello, nice to meet you. The experiments you made were really great and I want to find the axon tracks in human brain just like your elife paper said. I chose the `GW24` dataset you reconstructed using QLIPP and two-step algorithm. After background corrected, I chose 1250 pieces of images and saprated these 2048 * 2048 pixel pictures into 256 * 256 pixel without resize, and I also made z-score standardization and `Otsu` thresholding(`Rosin` is also attempted in another experiment). Then I trained images using 2DUnet and it came out with the PCC during training about 0.80+. However in inference stage, the result was not good, and I can not find the axon track just as clear as fluorescence image. Could you help me find how to solve my problem? Is that because 256 * 256 pixel images are two big or two small, or the deep learning model I chose are easily overfitting or just can't fit this complex nonlinear transformation from label-free to fluorescence?

**Comment by mattersoflight** [comment]
@yingmuzhi glad to hear that our data and code have been helpful. I suspect the problem you are seeing is due to normalization per FOV. When virtual staining across a large structure, it is important to normalize across the whole dataset. See this result: https://elifesciences.org/articles/55502/figures#fig7s1. There is a corresponding paragraph in the methods section of the paper that describes the details of the normalization - the main detail is that one should use median and inter-quartile range, rather than mean and standard deviation for z-scoring the dataset.

If you post your config and some example images, we can comment further.

**Comment by ziw-liu** [comment]
@yingmuzhi I can't access the files you linked. Can you double check that they are valid?

**Comment by yingmuzhi** [comment]
@ziw-liu hello, doctor Liu, I tried copy the url like `https://github.com/yingmuzhi/issues/blob/master/issue01/preprocess.yml` in the search bar, and then it will show up, not click directly.
